include/EvolutionNet/EvolutionNet.hpp

```diff
@@ -51,10 +52,12 @@ class EvolutionNet {
 
   /*! \brief Initialize the Evolution Net.
    *  You need to call this function before any other operation.
-   *  Here you can set the size of the population and the random seed used internally.
-   *  \note `populationSize` must be greater than zero, otherwise Undefined Behavior.
+   *  Here you can set the size of the population and the random seed used
+   * internally. \note `populationSize` must be greater than zero, otherwise
```

Comment:
are we able to keep indentation for doxygen comments?

* Indentation should be aligned with the first line
* `\command` should be prefixed by newline